CONTRIBUTING.md

```diff
@@ -13,6 +13,7 @@ This section describes the main steps required before you can add your code and
 ### Prerequisites
 * Linux machine with x86-64 architecture with bash installed.
 * Basic tools: curl, tar, make.
+* Python3 with pipenv package.
```

Comment:
I think we will also need the step of syncing with the lock file. Need to see if this can be done as part of the makefile in a way that can be run multiple times and work correctly and not make execution take too long.

Reply:
> Need to see if we will also need the step of syncing with the lock file. Need to see if this can be done as part of the makefile in a way that can be run multiple times and work correctly and not make execution take too long.

Since this prerequisite is not necessary at the moment, I removed it from the contributing guide and updated it.